src/contexts/archiveContext.js

```diff
@@ -0,0 +1,70 @@
+import { createContext, useContext, useEffect } from "react";
+import { useAuth, useNotes } from "contexts";
+import {
+  getArchivedService,
+  archiveNoteService,
+  unArchiveNoteService,
+} from "services";
+
+const ArchiveContext = createContext();
+
+const ArchiveProvider = ({ children }) => {
+  const { isAuth, token } = useAuth();
+  const { dispatchNote } = useNotes();
+
+  useEffect(() => {
+    if (isAuth) {
+      (async () => {
+        const { data, status } = await getArchivedService(token);
+
+        if (status === 200) {
+          dispatchNote({ type: "SET_ARCHIVED", payload: data.archives });
+        }
+      })();
+    }
+  }, [token]);
+
+  const archiveNote = async (e, note) => {
+    e.stopPropagation();
+
+    try {
+      const { data, status } = await archiveNoteService(note, token);
+
+      if (status === 201) {
+        dispatchNote({
+          type: "SET_NOTES_AND_ARCHIVE",
+          payload: { notes: data.notes, archives: data.archives },
+        });
+      }
+    } catch (err) {
+      console.error(err);
+    }
```

Comment:
suggestion: don't handle your error with the console. you can use any other tool like toast na

Reply:
Yes, I'll add toasts later. Thanks.